.gitignore

```diff
@@ -14,16 +14,25 @@
 ##########
 # Addons files
 ##########
+!/addons
```

Comment:
I can't add plugins that are different from `dialog_plugin` or related to it in this repository (due this is only to store this plugin and all related files to make it works without problems, like the unit test framework)

Reply:
If you want to add a specific folder, add it here in the `.gitignore` file. I need to be explicit about what is going to be in the repository only

Reply:
Oh yeah, this wasn't intentional - thought I had to exclude each folder but it wouldn't necessarily mean the folder contents would be included.